azure-servicebus/src/test/java/com/microsoft/azure/servicebus/TestCommons.java

```diff
@@ -507,6 +508,59 @@ public static void testReceiveBySequenceNumberAndDeadletter(IMessageSender sende
 			// Expected
 		}
 	}
+	
+	public static void testSendReceiveMessageWithVariousPropertyTypes(IMessageSender sender, String sessionId, IMessageReceiver receiver) throws InterruptedException, ServiceBusException
+	{
+		Map<String, Object> sentProperties = new HashMap<>();
```

Comment:
>Map<String, Object> sentProperties = new HashMap<>(); [](start = 2, length = 53)

Could you add this to javadoc of IMessage.java with the list of property types which are supported?